subsys/caf/modules/power_manager.c

```diff
@@ -142,25 +145,47 @@ static void system_off(void)
 	}
 }
 
+static void system_off_post_action(void) {
+
+/* You shouldn't enable CONFIG_POWEROFF and CONFIG_PM simultaneously */
+#if defined CONFIG_POWEROFF && defined CONFIG_PM
+		BUILD_ASSERT(false, "Redundant power configs turned on");
+#endif
+
+		if (IS_ENABLED(CONFIG_PM)) {
+		/* On nRF54 series, the poweroff is not supported yet. */
+		LOG_WRN("System suspended, the poweroff is not supported yet on nrf54h20");
```

Comment:
Can we keep the information specific to nRF54H only in the comment (alternatively, you could make an additional conditional log - displaying this information for nRF52 SoCs would be confusing)